specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Commands.json

```diff
@@ -96,6 +96,59 @@
     "MigrateSyncCompleteCommandOutput": {
       "type": "object",
       "description": "Output for command that completes sync migration for a database.",
+	  "properties": {
+        "id": {
+          "type": "string",
+          "description": "Result identifier",
+          "readOnly": true
+        },
+        "errors": {
+          "description": "List of errors that happened during the command execution",
+          "type": "array",
+          "items": {
+            "$ref": "./Common.json#/definitions/ReportableException"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "MigrateMISyncCompleteCommandProperties": {
+      "x-ms-discriminator-value": "Migrate.SqlServer.AzureDbSqlMi.Complete",
+      "type": "object",
+      "description": "Properties for the command that completes sync migration for a managed instance database.",
```

Comment:
"Azure SQL Database Managed Instance"

Reply:
"online migration"

Reply:
I think we are removing online and keeping sync everywhere